.gitignore

```diff
@@ -0,0 +1 @@
+*.beam
```

Comment:
No new line at the end of file

Reply:
Done!, i added a new line at the of the file and change the name of the PR.